backend/core/src/email/email.service.spec.ts

```diff
@@ -30,6 +33,18 @@ const application = {
   confirmationCode: "abc123",
 }
 let sendMock
+const httpPostMockFn = jest.fn()
+const mockHttpService = {
+  post: httpPostMockFn.mockImplementation(() => {
+    return of({
```

Comment:
Do you mind explaining what of() is doing here?

Reply:
The httpService returns an rxjs observable. This "of" function converts the passed in params to an observable. https://rxjs.dev/api/index/function/of. We don't actually care about the data that is returned, so I could put anything in here. But it needs to be of type observable